utilities/backup/backup_engine_test.cc

```diff
@@ -2696,7 +2703,6 @@ TEST_P(BackupEngineRateLimitingTestWithParam, RateLimitingVerifyBackup) {
   auto rate_limited_verify_backup_time =
       (bytes_read_during_verify_backup * kMicrosPerSec) /
       backup_rate_limiter_limit;
-
```

Comment:
Potential merge conflict with https://github.com/facebook/rocksdb/pull/9974, depending on which one of us merges first - shouldn't be a tough one though

Reply:
Thanks for the heads up!